.github/workflows/python-package.yml

```
@@ -78,7 +78,7 @@ jobs:
         mkdir ~/blob_emulator
         azurite -l ~/blob_emulator --debug debug.log 2>&1 > stdouterr.log &
         pytest --cov=zarr --cov-config=pyproject.toml --doctest-plus --cov-report xml --cov=./ --timeout=300
-    - uses: codecov/codecov-action@v3
+    - uses: codecov/codecov-action@v4
       with:
         token: ${{ secrets.CODECOV_TOKEN }} # not required for public repos
```

With `v4`, the token is required by all repositories, but it is not passed down to the forks.
```suggestion
        token: ${{ secrets.CODECOV_TOKEN }}
```